addons/edi/models/edi_document.py

```diff
@@ -85,6 +85,11 @@ def _default_project_id(self):
     _sql_constraints = [('model_uniq', 'unique (model_id)',
                          "The document model must be unique")]
 
+    # optionally enforce document name globs
+    enforce_filename = fields.Boolean(string="Enforce document filenames follow glob pattern",
```

Comment:
We probably want to add this field to the view so that and admin can turn it on and off

Reply:
Jumping on this, probably the string is a bit too long here too... I think `Enforce Document Filenames` is enough, the `help` string can go into more detail.

Reply:
String looks good, to add the field to the view we can just add it here: https://github.com/unipartdigital/odoo-edi/blob/master/addons/edi/views/edi_document_type_views.xml#L6 - either in the basic or extras pages.

Also small nit: Could you move this field above `_sql_constraints`? i.e. put it under `fail_fast`. :)